scripts/aws/promote-ami.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+if env | grep -q "NYC_TREES_DEPLOY_DEBUG"; then
+  set -x
+fi
+
+function packer_isotime() {
+   date -u +"%Y-%m-%dT%H:%M:%SZ"
+}
+
+if (env | grep -q "NYC_TREES_PROMOTE_AMI" && env | grep -q "NYC_TREES_PROMOTE_ACCOUNT_ID"); then
```

Comment:
So in the scenario where we are deploying an AMI from staging to production, `NYC_TREES_PROMOTE_AMI` represents an AMI ID that exists on the staging account (or whatever account is configured as `AWS_PROFILE`) and `NYC_TREES_PROMOTE_ACCOUNT_ID` represents the AWS User ID for the production account?